tasks.py

```diff
@@ -81,8 +90,22 @@ def main():
 
         operation, *args = line.split()
         fn = operations.get(operation)
-        fn(*args)
+
+        if operation == 'create':
+            args = " ".join([*args])
```

Comment:
This should be the same as `" ".join(args)`, no need to make a list and unpack the tuple into it.